salt/_pillar/metalk8s_private.py

```diff
@@ -28,6 +30,23 @@ def _read_sa_private_key():
     return {"sa_private_key": key_data}
 
 
+def _read_apiserver_key():
+    # FIXME: we only have access to this file because:
+    #   1- Salt master Pod mounts all of /etc/kubernetes
+    #   2- Salt master Pod runs on the 'bootstrap' minion, which is, by chance,
+    #      also the initial master node
+
+    if not os.path.isfile(APISERVER_KEY_PATH):
```

Comment:
TOCTOU:, use a try/except (you can have two except, one for not found and one for the others) below instead.

Reply:
cargo culting is bad, mmmkay?